src/common/Button/Button.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonText, onClick, value }) => (
+const Button = ({ children, buttonText, onClick, value }) => (
 	<button value={value} onClick={onClick}>
-		{buttonText}
+		{children ? children : buttonText}
```

Comment:
this would be good to use it like:  { children ?? buttonText } 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Thanks good approach